tests/unit_tests/feature_selection/test_feature_selectors.py

```diff
@@ -35,7 +35,7 @@ def tearDown(self) -> None:
     def validate_feature_selector(self, feature_selector, **kwargs):
         df = copy.deepcopy(self.features_dataset)
 
-        def side_effect(arg):
+        def side_effect(arg, inplace=True):
```

Comment:
ok, this should be related with the change in the feature_selection module, still i don't fully understand why...